Procfile

```diff
@@ -0,0 +1 @@
+web: node app.js
```

Comment:
why is a Procfile being added?

Reply:
For heroku deployment

Reply:
It is not required from my experiments.  It is out of the scope of "Sign in with Slack" as well, so let's remove it.

Reply:
ok